internal/apis/certmanager/types.go

```diff
@@ -183,3 +183,9 @@ const (
 	UsageMicrosoftSGC      KeyUsage = "microsoft sgc"
 	UsageNetscapeSGC       KeyUsage = "netscape sgc"
 )
+
+const (
+	// CleanupPolicy values
+	CleanupPolicyNever  = "Never"
+	CleanupPolicyDelete = "Delete"
+)
```

Comment:
In order to conform with the rest of how the Kubernetes codebase does, let's add a type alias for this:

```suggestion
type CleanupPolicy string

const (
	// CleanupPolicy values
	CleanupPolicyNever CleanupPolicy = "Never"
	CleanupPolicyDelete CleanupPolicy = "Delete"
)
```

then use the type `CleanupPolicy` below.